config/jobs/kubernetes/sig-windows/windows-gce.yaml

```diff
@@ -581,6 +581,10 @@ presubmits:
           value: "win2019"
         - name: PREPULL_YAML
           value: "prepull-head.yaml"
+        # Disable unit test for Windows for now.
+        # TODO: Should remove it after unit tests are fixed.
```

Comment:
What is broken about the unit tests? Is there an issue tracking this? Will they ever be fixed, or will this never work for Windows?

Reply:
1. they're broken on windows, there is an issue in k/k
2. when running e2e we should not be running the unit tests
3. these run when building a release by default. they do not when running a quick release